### ISSUE TITLE
Feature to hide pinyin

### ISSUE THREAD

**Issue by caeldom** [cschiller/zhongwen#37]
Hi, I have been using this extension for ages, thank you!

I'd like to request a feature to be able to hide the pinyin, for example, when Zhuyin is turned on. It is helpful for learning and memorisation.

**Comment by cschiller** [comment]
Hi, I'm trying to understand this feature request. So you're trying to learn Zhuyin and don't want to see Pinyin? Wouldn't it help to see Pinyin in this case? Othewise how would you know whether you've got the Zhuyin right?

**Comment by caeldom** [comment]
Hi, I can read Zhuyin (without any aid), so I use it as an alternative to Pinyin because I prefer not to have any romanisation at all, if that makes more sense.

**Comment by cschiller** [comment]
I see, but isn't the goal to read the Hanzi without any Pinyin or Zhuyin?

**Comment by cschiller** [comment]
But I don't understand, if you look up words that you don't know, how would you know how they're pronounced if you see neither Pinyin nor Zhuyin?

**Comment by caeldom** [comment]
I would have Zhuyin turned on by default. Just don't want to see the Pinyin.

**Comment by marklalor** [comment]
Can second that hiding pinyin can be helpful. It can help you focus that the sounds are totally unrelated to the latin letters. Pinyin with respect to the phonetics has a few one-to-many relationships that can make it hard for new learners. Examples:

wu, yu, etc don't have initial sounds. This is particularly confusing for cases like yao (iao) or ya (ia) because learners don't realize this is the same sound as in jiao, jia, etc.

yuan, qu, etc are confusing one because it's the same ü sound as in nü. This is the only one that I don't understand, why didn't the pinyin creators make these yüan and qü...?

ri has no final while something like li does. Definitely said "ree" or "lrrr" in my head a few times while learning 😅avoiding situations like that would be a good reason to have this feature.

Also I think Taiwan uses Zhuyin over Pinyin so I think that's another good reason.

**Comment by cschiller** [comment]
Zhongwen uses [CC-CEDICT](https://cc-cedict.org/wiki/start) as its built-in dictionary. CC-CEDICT is based on Pinyin. The Zhuyin you see in Zhongwen is automatically generated. I have no idea whether - and cannot guartantee that - the Zhuyin  shown is entirely accurate. For this reason I do not want to hide Pinyin. (By the way, sometimes the Pinyin is also incorrect. The tones are sometimes wrong, for instance. But this is something that the CC-CEDICT editors need to address.)

Even if you only know Zhuyin, I think also learning Pinyin would be tremendously helpful. Like it or not, it is the de-facto transliteration standard for Mandarin Chinese.

**Comment by caeldom** [comment]
Yes, I understand learning Pinyin is helpful. However for advanced learners who want to learn the Chinese character without seeing any English, it is extremely distracting and unhelpful for memorisarion to see the pronunciation show up every time. We only need to see it once and after that have the option to hide it. It is like using flashcards. If you do not want to hide it, then at least have an option to make it smaller or less obvious.

**Comment by dpwright** [comment]
Came here to make the same request and found the issue already exists.  I realise that pinyin is the standard, but for those that want to use zhuyin it would be much nicer if we could hide the pinyin. For me, it's because I can't *not* read the pinyin if it's there -- the Latin characters jump out at me whether I want them to or not -- so if I'm just checking the meaning and don't want to be faced with the pronunciation I can't as easily ignore it. I also prefer zhuyin generally, for all the reasons stated above by @marklalor. The wisdom of this can be debated, and from what I've seen online it is debated, endlessly, but I am not trying to make the argument that one system is better than the other; just that for those who -- whatever their reasons may be -- prefer not to see pinyin, it would be nice to have the option to hide it.

Your concern about the zhuyin being automatically generated and potentially incorrect is certainly valid.  Perhaps you could put a big red warning next to the option pointing this danger out to the user?

**Comment by caeldom** [comment]
> I have no idea whether - and cannot guartantee that - the Zhuyin shown is entirely accurate. For this reason I do not want to hide Pinyin.
You can have this as a warning next to the checkbox option for Only show Zhuyin, that way users will be aware if they so choose to hide the pinyin

**Comment by Garbaz** [comment]
Until this feature gets implemented in the extension itself, for anyone interested I've written a [userscript](https://greasyfork.org/en/scripts/429193-zhongwen-zhuyin-only-font-change) that simply adds some global CSS which hides pinyin in the Zhongwen pop-up (and also changes the font, since I'm personally not a fan of the cursive one).

**Comment by KrulYuno** [comment]
So, has this feature been implemented? I do not see the option to disable pinyin.
